tests/objc-cli/libmanaged/Tests/Tests.m

```diff
@@ -657,8 +657,8 @@ - (void) testIsEqual {
     XCTAssertTrue ([c1 isEqual:c1], "equals self");
 
     EqualsHashOverrides_Class *c2 = [[EqualsHashOverrides_Class alloc] initWithX:1];
-    XCTAssertTrue ([c1 isEqual:c2], "compare equal objects");
-    XCTAssertTrue ([c2 isEqual:c1], "compare equal objects");
+    XCTAssertTrue ([c1 areEqual:c2], "compare equal objects");
```

Comment:
Mmh, maybe I didn't understand https://github.com/mono/Embeddinator-4000/issues/221#issue-225436572 but wasn't `areEqual` supposed to be used for the static version?

Because the naming is wrong here.
It should remain: `[c1 isEqual:c2]` or even better `[c1 isEqualTo EqualOverrides:c2]`